wavefront_python_sdk/direct_ingestion/wavefront_direct_ingestion_client.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import absolute_import, division, print_function
+
+from wavefront_python_sdk.common.connection_handler import IConnectionHandler
+from wavefront_python_sdk.common.utils import metric_to_line_data, \
+    histogram_to_line_data, tracing_span_to_line_data
+
+import sys
+import requests
+from threading import Timer
+from queue import Queue
+
+
+class WavefrontDirectIngestionClient(IConnectionHandler):
+    WAVEFRONT_METRIC_FORMAT = 'wavefront'
+    WAVEFRONT_HISTOGRAM_FORMAT = 'histogram'
+    WAVEFRONT_TRACING_SPAN_FORMAT = 'trace'
+
+    def __init__(self, server, token, max_queue_size=50000, batch_size=10000, flush_interval_seconds=1):
```

Comment:
docstring for the constructor. Clearly specify the format for server: `https://INSTANCE.wavefront.com`and specify direct data ingestion permission is required for the token.